spec/requests/whitehall_media_requests_spec.rb

```diff
@@ -23,7 +23,7 @@
     end
 
     it 'redirects to placeholder image' do
-      expect(response).to redirect_to('/images/thumbnail-placeholder.png')
+      expect(response).to redirect_to(Regexp.new('/asset-manager/thumbnail-placeholder-.*\.png'))
```

Comment:
Minor: I might prefer the regular expression to further constrain the path to *start* with `/asset-manager`.